docs/docs/98-contributing/02-tests.md

```diff
@@ -0,0 +1,53 @@
+# Tests and linting
+
+Mapperly is continuously tested by GitHub Actions.
+Tests are separated into integration and unit tests located in the `test` directory
+and use [xUnit](https://xunit.net/) and [VerifyTests](https://github.com/VerifyTests/Verify).
+You can run the tests by running
+```bash
+dotnet test
+```
+or by running the discovered tests in your IDE.
+
+## Unit tests
+
+The unit tests are located in the `tests/Riok.Mapperly.*.Tests` projects.
+These unit tests usually test exactly one unit of code or one type of mapping by isolating it's code
+and verifying the reported diagnostics and the emitted code.
+Unit tests are easy to debug (you can debug them like any other code),
+but be reminded that these kind of tests only run on the latest supported target framework.
+
+## Integration tests
+
+The integration tests are located in `tests/Riok.Mapperly.IntegrationTests`.
+Integration tests are "a bit of everything" mappers.
+These tests run locally by referencing the source generator as an analyzer.
+In the CI pipeline, the integration tests reference the built NuGet package and
+are run on several supported target frameworks (including .NET 7.0 but also .NET Framework).
+
+Debugging integration tests is a lot harder than debugging unit tests.
+Therefore if a integration test needs to be debugged,
+it is often easier to implement an unit test for the to be tested behaviour
+and debug the unit test instead of the integration test.
+See also the [debugging](./debugging) documentation topic on how to debug Mapperly.
```

Comment:
Would include "documentation" in the link (currently it is outside the link)

Reply:
Improved